bodhi/server/services/updates.py

```diff
@@ -20,6 +20,7 @@
 import copy
 import math
 
+from bodhi_messages.schemas import update as update_schemas
```

Comment:
I think this import still should be at line 64. any reasons it was moved up ?

Reply:
Because when it was `bodhi.messages` it was identified as a local package, but now `bodhi_messages` is identified as Third Party.

Reply:
Oh makes sense :+1: